extensions/amp-carousel/0.1/test/test-slides.js

```diff
@@ -920,7 +920,7 @@ describe('empty Slides functional', () => {
     }).to.throw(/should have at least 1 slide/);
     expect(() => {
       slides.layoutCallback();
-    }).to.not.throw();
+    }).to.throw(/Element expected/);
```

Comment:
it makes sense here since there is no element (no slides, so why bother?) to lay out.